waspc/cli/exe/Main.hs

```diff
@@ -92,6 +92,9 @@ printUsage =
         cmd "    new <project-name>    Creates new Wasp project.",
         cmd "    version               Prints current version of CLI.",
         cmd "    waspls                Run Wasp Language Server. Add --help to get more info.",
+        title "  COMPLETION",
```

Comment:
Maybe let's just move it under GENERAL, I don't think we need special title for it.
Also, I wonder if we should skip `completion:generate`? I believe that one is elaborated upon in `completion` command. Just to make it a bit less crowded here, I don't think we want completion taking too much focus.